.github/workflows/lint-php.yml

```diff
@@ -66,7 +66,7 @@ jobs:
         run: composer install --prefer-dist --no-progress --no-interaction
 
       - name: Detect coding standard violations (PHPCS)
-        run: vendor/bin/phpcs -q --report=checkstyle --runtime-set ignore_errors_on_exit 1 --runtime-set ignore_warnings_on_exit 1 | cs2pr --graceful-warnings
+        run: vendor/bin/phpcs -q --report=checkstyle --severity=1 --runtime-set ignore_errors_on_exit 1 --runtime-set ignore_warnings_on_exit 1 | cs2pr --graceful-warnings
```

Comment:
Why level 1? Could this / should this to be higher?

Reply:
By default, PHPCS assigns a severity of 5 to all errors and warnings. That means any messages with severity 1-4 are _not_ reported. By specifically adding `severity=1` we see _more_ messages.

If we would make this higher then we would not see any messages at all.